### ISSUE TITLE
Improve design

### ISSUE THREAD

**Issue by Michael55555** [Michael55555/discord-bot-creator#2]
There are still a couple of design issues that need to be fixed.
- [ ] Remove sidebar scroll
- [ ] Make input fields same length

**Comment by belamajoros** [comment]
Which sidebar scroll needs to be removed? One appears when the window height is <361 and a second one when window height is <164.

**Comment by m5x5** [comment]
We need to remove the sidebar that appears when there are too many commands or events. It makes the sidebar larger instead of adding a scroll bar just to the list.

**Comment by m5x5** [comment]
> Which sidebar scroll needs to be removed? One appears when the window height is <361 and a second one when window height is <164.

I believe I now understood your question here. Yes, we have another scrollbar that appears when the window width is just < 450 or something about that number. Would you like to remove that one too? @flambi99

**Comment by belamajoros** [comment]
Is this what u mean? https://imgur.com/a/JmYMctR

**Comment by m5x5** [comment]
Yes, this one too. Sorry for not being concise. This one is what I initially meant, the second scroll bar shouldn't be there
![image](https://user-images.githubusercontent.com/30301026/137589656-b84a9c31-ed68-4f7d-a0ce-8583fbe5edec.png)

**Comment by belamajoros** [comment]
Okay i'll look into it later :)

**Comment by m5x5** [comment]
Thank you!
